config.md

```diff
@@ -128,27 +131,25 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001's `environ`][ieee-1003.1-2001-xbd-c8.1].
 * **`args`** (array of strings, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execvp`'s *argv*][ieee-1003.1-2001-xsh-exec].
   This specification extends the IEEE standard in that at least one entry is REQUIRED, and that entry is used with the same semantics as `execvp`'s *file*.
-
-For Linux-based systems the process structure supports the following process specific fields:
-
-* **`capabilities`** (array of strings, OPTIONAL) capabilities is an array that specifies Linux capabilities that can be provided to the process inside the container.
-  Valid values are the strings for capabilities defined in [the man page](http://man7.org/linux/man-pages/man7/capabilities.7.html).
+* **`capabilities`** (array of strings, OPTIONAL) is an array that specifies the set of capabilities of the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [CAPABILITIES(7)](http://man7.org/linux/man-pages/man7/capabilities.7.html) man page.
```

Comment:
I am having difficulty drawing that conclusion. As far as I know, there is no requirement for a minimum set of platform-specific configuration fields to be populated. This change doesn't seem to undo anything, and in fact corrects an error. For this specific field, I've changed the statement that valid values for all platforms are documented in a Linux man page, to a statement which says that valid values are platform-specific, and valid values must be used for this field. It even uses the same man page as an example. I am fine with claiming that a minimum amount of functionality must be implemented, but I don't see how that has bearing on this set of changes. I do not see how this introduces an argument that we avoid or weaken runtime compliance testing or certification.